diff --git a/xmlsecurity/Module_xmlsecurity.mk b/xmlsecurity/Module_xmlsecurity.mk
index 17efc609d27a..c01aa6fb6f3d 100644
--- a/xmlsecurity/Module_xmlsecurity.mk
+++ b/xmlsecurity/Module_xmlsecurity.mk
@@ -11,8 +11,6 @@
 
 $(eval $(call gb_Module_add_targets,xmlsecurity,\
 	Library_xmlsecurity \
-	$(if $(ENABLE_NSS)$(ENABLE_OPENSSL),\
-	    Library_xsec_xmlsec \
-	    UIConfig_xmlsec \
-	) \
+	$(if $(ENABLE_NSS)$(ENABLE_OPENSSL)$(filter WNT,$(OS)),Library_xsec_xmlsec) \
+	$(if $(ENABLE_NSS)$(ENABLE_OPENSSL),UIConfig_xmlsec) \
 ))
