--- a/sc/source/core/data/formulacell.cxx
+++ b/sc/source/core/data/formulacell.cxx
@@ -5139,7 +5139,7 @@ bool ScFormulaCell::InterpretFormulaGroup()
 
     int nMaxGroupLength = INT_MAX;
 
-#ifdef _WIN32
+#if HAVE_FEATURE_OPENCL && defined(_WIN32)
     // Heuristic: Certain old low-end OpenCL implementations don't
     // work for us with too large group lengths. 1000 was determined
     // empirically to be a good compromise.
