--- a/sd/source/ui/inc/View.hxx
+++ b/sd/source/ui/inc/View.hxx
@@ -39,7 +39,9 @@ class ImageMap;
 class Graphic;
 class SdrOutliner;
 
+#if HAVE_FEATURE_AVMEDIA
 namespace avmedia { class PlayerListener; }
+#endif
 
 namespace sd {
 
@@ -268,7 +270,9 @@ protected:
     sal_Int8                mnAction;
     Idle                    maDropErrorIdle;
     Idle                    maDropInsertFileIdle;
+#if HAVE_FEATURE_AVMEDIA
     rtl::Reference<avmedia::PlayerListener> mxDropMediaSizeListener;
+#endif
     sal_uInt16              mnLockRedrawSmph;
     bool                    mbIsDropAllowed;
 
--- a/sd/inc/pch/precompiled_sd.hxx
+++ b/sd/inc/pch/precompiled_sd.hxx
@@ -125,8 +125,10 @@
 #include <vcl/wrkwin.hxx>
 #endif // PCH_LEVEL >= 2
 #if PCH_LEVEL >= 3
+#if HAVE_FEATURE_AVMEDIA
 #include <avmedia/mediaplayer.hxx>
 #include <avmedia/mediawindow.hxx>
+#endif
 #include <basegfx/basegfxdllapi.h>
 #include <basegfx/color/bcolor.hxx>
 #include <basegfx/matrix/b2dhommatrix.hxx>
