diff --git a/src/md4c.c b/src/md4c.c
--- a/src/md4c.c
+++ b/src/md4c.c
@@ -343,11 +343,11 @@ struct MD_VERBATIMLINE_tag {
 #define ISANYOF2_(ch, ch1, ch2)         ((ch) == (ch1) || (ch) == (ch2))
 #define ISANYOF3_(ch, ch1, ch2, ch3)    ((ch) == (ch1) || (ch) == (ch2) || (ch) == (ch3))
 #define ISASCII_(ch)                    ((unsigned)(ch) <= 127)
-#define ISBLANK_(ch)                    (ISANYOF2_((ch), _T(' '), _T('\t')))
+#define ISBLANK_(ch)                    (ISANYOF2_((ch), _T(' '), _T('\t')) || (unsigned)(ch) == 0x3000)
 #define ISNEWLINE_(ch)                  (ISANYOF2_((ch), _T('\r'), _T('\n')))
 #define ISWHITESPACE_(ch)               (ISBLANK_(ch) || ISANYOF2_((ch), _T('\v'), _T('\f')))
 #define ISCNTRL_(ch)                    ((unsigned)(ch) <= 31 || (unsigned)(ch) == 127)
-#define ISPUNCT_(ch)                    (ISIN_(ch, 33, 47) || ISIN_(ch, 58, 64) || ISIN_(ch, 91, 96) || ISIN_(ch, 123, 126))
+#define ISPUNCT_(ch)                    (ISIN_(ch, 33, 47) || ISIN_(ch, 58, 64) || ISIN_(ch, 91, 96) || ISIN_(ch, 123, 126) || ISIN_(ch, 0x3001, 0x303F) || ISIN_(ch, 0xFF01, 0xFF65))
 #define ISUPPER_(ch)                    (ISIN_(ch, _T('A'), _T('Z')))
 #define ISLOWER_(ch)                    (ISIN_(ch, _T('a'), _T('z')))
 #define ISALPHA_(ch)                    (ISUPPER_(ch) || ISLOWER_(ch))
